HISTORY.md

```diff
@@ -9,6 +9,7 @@
 ### New Features
 * DB::GetLiveFilesStorageInfo is ready for production use.
 * Add new stats PREFETCHED_BYTES_DISCARDED which records number of prefetched bytes discarded by RocksDB FilePrefetchBuffer on destruction and POLL_WAIT_MICROS records wait time for FS::Poll API completion.
+* Start tracking SST unique id in MANIFEST, which will be used to verify with SST properties during DB open. A db option `try_verify_sst_unique_id` is introduced to enable/disable the verification.
```

Comment:
(1) It might be good to explain why we are introducing this feature (copy the info from the PR's summary)
(2) also, state this feature is disabled on default

Reply:
added more comments.